.storybook/preview.tsx

```diff
@@ -9,19 +9,7 @@ import 'focus-visible/dist/focus-visible';
 
 initializeWorker();
 
-export const decorators = [
-	mswDecorator,
-	withDatxProvider,
-	withThemeProvider,
-	withPerformance,
-	withNextRouter({
```

Comment:
You removed https://github.com/lifeiscontent/storybook-addon-next-router plugin.
Is this now supported OOB in storybook?

Reply:
After updating `storybook-addon-next-router` plugin to newer version I followed this tutorial for the setup - https://storybook.js.org/addons/storybook-addon-next-router/. 
I now see that I didn't add `storybook-addon-next-router` to the `.storybook/main.js`. I will add it immediately.